src/styles/components/_value.scss

```diff
@@ -35,10 +37,11 @@
     display: flex;
     justify-content: space-between;
     opacity: 0;
-
-    transition: opacity 0.3s;
+    transform: translateX(-100%);
```

Comment:
avoid such a big values
better to use smth like transform(-100px) to see smooth animation
if you want to hide element completely do the trick with `visibility: hidden` state

https://css-tricks.com/snippets/css/toggle-visibility-when-hiding-elements/